pom.xml

```diff
@@ -807,9 +807,56 @@ under the License.</licenseText>
             </dependency>
           </dependencies>
         </plugin>
+        <plugin>
+          <groupId>org.openrewrite.maven</groupId>
+          <artifactId>rewrite-maven-plugin</artifactId>
+          <version>6.8.1</version>
+          <configuration>
+            <activeRecipes>
+            <recipe>org.openrewrite.java.format.AutoFormat</recipe>
```

Comment:
ideally this could supplement, substitute, might even replace, `spot` and `check`.

Reply:
Yes might see at the end they must be equal having to obey same convention. 

This is just optional. Spot and openrewrite in general a 2 of a kind. Spot ist just file parser. Rewrite will cure all.
Just see the PR´s we have friction because of not ensure code standard.